architecture-evaluation/src/main/java/edu/kit/kastel/sdq/case4lang/refactorlizar/architecture_evaluation/graphs/SystemGraphUtils.java

```diff
@@ -0,0 +1,11 @@
+/** */
+package edu.kit.kastel.sdq.case4lang.refactorlizar.architecture_evaluation.graphs;
+
+import com.google.common.graph.Graph;
+import com.google.common.graph.MutableGraph;
+
+/** @author Reiner Jung */
```

Comment:
warning: [MissingSummary] A summary line is required on public/protected Javadocs.
/** @author Reiner Jung */
    ^
    (see https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment)

Reply:
Added a comment.